README.md

```diff
@@ -41,7 +41,7 @@ To add inflector via `web.xml`:
   <servlet-class>org.glassfish.jersey.servlet.ServletContainer</servlet-class>
   <init-param>
     <param-name>javax.ws.rs.Application</param-name>
-    <param-value>io.swagger.inflector.SwaggerInflector</param-value>
+    <param-value>SwaggerInflector</param-value>
```

Comment:
I think the class for this example is: `io.swagger.oas.inflector.OpenAPIInflector`